android/app/src/main/java/com/zulipmobile/sharing/SharingHelper.kt

```diff
@@ -25,16 +25,16 @@ fun maybeHandleIntent(
     contentResolver: ContentResolver
 ): Boolean {
     // We handle intents from "sharing" something to Zulip.
-    if (intent?.action == Intent.ACTION_SEND) {
-        handleSend(intent, application, contentResolver)
-        return true
+
+    when (intent?.action) {
+        Intent.ACTION_SEND -> handleSend(intent, application, contentResolver)
+        Intent.ACTION_SEND_MULTIPLE -> handleSend(intent, application, contentResolver)
+        // For other intents, let RN handle it.  In particular this is
+        // important for VIEW intents with zulip: URLs.
+        else -> return false
```

Comment:
The cases we handle should continue to return true, as the doc comment says we will.

Reply:
... Oh I see, it does that below.

That's confusing -- at least, evidently I was confused :wink:. Better to pair the handling of the intent with the returning of true to say we did so.